research-software-directory/RSD-as-a-service
# Save ROR data to RSD and use it

# Save ROR data in RSD

Closes #1333 

Changes proposed in this pull request:
* ROR scraper updates organisation data
* We save more organisation data to RSD (city, wikipedia_url, lat,lon etc.) and use these instead of making requests to ROR api. 

How to test:
* it is better to use empty RSD for this test, so use `docker compose build && docker compose up`
* login to RSD
* create one software entry.
* add eScienceCenter organisation from ROR to software
* navigate to organisation and confirm that ROR data is visible
* create one project entry
* add Microsoft organisation from ROR to project
* navigate to organisation and confirm that ROR data is visible
* now add eScienceCenter organisation to project. This should work too, but now the organisation is added from RSD
* add third organisation manually: search for organisation like "New organisation I want to add". The option "Add..." should appear so you can add organisation manually.
* navigate to this new organisation. This organisation should not have ROR information (ROR link, type etc).
* examine console logs and confirm that ROR scraper is running from time to time :-).

## Example ROR organisation page
![image](https://github.com/user-attachments/assets/0abafcd0-e53f-42b7-bb07-99ae6a16a7cf)


PR Checklist:
